toolchain/sem_ir/entity_with_params_base.h

```diff
@@ -95,21 +94,27 @@ struct EntityWithParamsBase {
   // A block containing the pattern insts for the parameter lists.
   InstBlockId pattern_block_id;
   // A block containing, for each implicit parameter, a reference to the
-  // instruction in the entity's declaration block that depends on all other
-  // pattern-match insts pertaining to that parameter.
-  InstBlockId implicit_param_refs_id;
-  // A block containing, for each implicit parameter, a reference to the
   // instruction in the entity's pattern block that depends on all other
   // pattern insts pertaining to that parameter.
   InstBlockId implicit_param_patterns_id;
   // A block containing, for each explicit parameter, a reference to the
-  // instruction in the entity's declaration block that depends on all other
-  // pattern-match insts pertaining to that parameter.
-  InstBlockId param_refs_id;
-  // A block containing, for each explicit parameter, a reference to the
   // instruction in the entity's pattern block that depends on all other
   // pattern insts pertaining to that parameter.
   InstBlockId param_patterns_id;
+  // If this entity is a function, this block consists of references to the
+  // `AnyParam` insts that represent the function's `Call` parameters. The
+  // "`Call` parameters" are the parameters corresponding to the arguments that
+  // are passed to a `Call` inst, so they do not include compile-time
+  // parameters, but they do include the return slot.
+  //
+  // The parameters appear in declaration order: `self` (if present), then the
+  // explicit runtime parameters, then the return slot (which is "declared" by
+  // the function's return type declaration). This is not populated on imported
+  // functions, because it is relevant only for a function definition.
+  //
+  // TODO: can this be moved to `Function`, since it is not applicable to other
```

Comment:
```suggestion
  // TODO: Can this be moved to `Function`, since it is not applicable to other
```

Capitalization for style (https://google.github.io/styleguide/cppguide.html#TODO_Comments)